Source/Mosa.Utility.Launcher/Starter.cs

```diff
@@ -226,40 +213,60 @@ private Process LaunchBochs(bool getOutput)
 			var logfile = Path.Combine(LauncherSettings.TemporaryFolder, Path.GetFileNameWithoutExtension(LauncherSettings.ImageFile) + "-bochs.log");
 			var configfile = Path.Combine(LauncherSettings.TemporaryFolder, Path.GetFileNameWithoutExtension(LauncherSettings.ImageFile) + ".bxrc");
 
-			var fileVersionInfo = FileVersionInfo.GetVersionInfo(LauncherSettings.Bochs);
-
-			// simd or sse
-			var simd = "simd";
-
-			if (!(fileVersionInfo.FileMajorPart >= 2 && fileVersionInfo.FileMinorPart >= 6 && fileVersionInfo.FileBuildPart >= 5))
-				simd = "sse";
-
 			var sb = new StringBuilder();
 
 			sb.AppendLine($"megs: {LauncherSettings.EmulatorMemory}");
-			sb.AppendLine("ata0: enabled=1,ioaddr1=0x1f0,ioaddr2=0x3f0,irq=14");
-			sb.AppendLine($"cpu: count={LauncherSettings.EmulatorCores}");
-			sb.AppendLine($"cpuid: mmx=1,sep=1,{simd}=sse4_2,apic=xapic,aes=1,movbe=1,xsave=1");
-			sb.AppendLine("boot: cdrom,disk");
+			sb.AppendLine($"ata0: enabled=1,ioaddr1=0x1f0,ioaddr2=0x3f0,irq=14");
+			sb.AppendLine($"cpuid: mmx=1,sep=1,simd=sse4_2,apic=xapic,aes=1,movbe=1,xsave=1");
+			sb.AppendLine($"boot: cdrom,disk");
 			sb.AppendLine($"log: {Quote(logfile)}");
 			sb.AppendLine($"romimage: file={Quote(Path.Combine(bochsdirectory, "BIOS-bochs-latest"))}");
 			sb.AppendLine($"vgaromimage: file={Quote(Path.Combine(bochsdirectory, "VGABIOS-lgpl-latest"))}");
-			sb.AppendLine($"display_library: x, options={Quote("gui_debug")}");
 
-			if (LauncherSettings.ImageFormat == "iso")
+			//sb.AppendLine($"vgaromimage: file={Quote(Path.Combine(bochsdirectory, "VGABIOS-lgpl-latest-cirrus"))}");
+
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+			{
+				sb.AppendLine($"display_library: x, options={Quote("gui_debug")}");
+			}
+
+			switch (LauncherSettings.ImageFormat)
+			{
+				case "iso": sb.AppendLine($"ata0-master: type=cdrom,path={Quote(LauncherSettings.ImageFile)},status=inserted"); break;
+				default: sb.AppendLine($"ata0-master: type=disk,path={Quote(LauncherSettings.ImageFile)},biosdetect=none,cylinders=0,heads=0,spt=0"); break;
+			}
+
+			switch (LauncherSettings.EmulatorSVGA)
+			{
+				case "vbe": sb.Append("vga: extension=vbe"); break;
+				case "cirrus": sb.Append("vga: extension=cirrus"); break;
+				default: break;
+			}
+
+			// COM1 = Kernel Log
+			if (LauncherSettings.LauncherTest)
 			{
-				sb.AppendLine($"ata0-master: type=cdrom,path={Quote(LauncherSettings.ImageFile)},status=inserted");
+				// Not supported
+				sb.AppendLine("com1: enabled=1, mode=null");
 			}
 			else
 			{
-				sb.AppendLine($"ata0-master: type=disk,path={Quote(LauncherSettings.ImageFile)},biosdetect=none,cylinders=0,heads=0,spt=0");
+				sb.AppendLine("com1: enabled=1, mode=null");
```

Comment:
Is this a mistake or is it intended? If it is intended, the if condition is useless.